### ISSUE TITLE
Could we support getting the html structure from the a11y snapshot?

### ISSUE THREAD

**Issue by chriszyyy** [microsoft/playwright-mcp#451]
My team is trying to use the tool to auto fix some UI bugs. 

First, we need to reproduce the bug from the repro steps explain in the bug item.
Then, we need to collect related CSS selectors from the target elements, so we could use these info to locate them in our code base.
Finally, we suggest a code fix with what we have reproduce and eventually verify automatically in our local env.

But the current Playwright MCP doesn't have the possibility to access the HTML content of the page or using the a11y ref to map it back to the DOM tree. Is there any plan of supporting such scenario?

**Comment by pavelfeldman** [comment]
No plans to expose DOM or HTML.
